packages/shared/src/types/metadata.ts

```diff
@@ -1,2 +1 @@
-// eslint-disable-next-line @typescript-eslint/no-empty-interface
-export interface Metadata {}
+export type Metadata = Record<string, string>;
```

Comment:
@lukas-reining do you remember why this was an empty interface?

Reply:
No, I think this was like this from the beginning, I think I just moved it when refactoring the SDK.
We should be able to just change it.